src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -83,6 +87,14 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.isFirefox();
   }
 
+  isMacOS(): boolean {
```

Comment:
We usually avoid create public platform APIs like this and instead create a wrapper on what the intention of the check is such as `needToSetLatencyForWebAudio`.

Reply:
Ack